netflix/src/api/getMovies.ts

```diff
@@ -0,0 +1,72 @@
+// now playing 영화 데이터 반환하는 함수.
+export async function getNowPlayingMovie() {
+  const url = `https://api.themoviedb.org/3/movie/now_playing?api_key=${process.env.NEXT_PUBLIC_MOVIE_API_KEY}`;
+  const nowPlayingMovieResponse = await fetch(url, { cache: 'no-store' });
+  
+  const nowPlayingMovieData = await nowPlayingMovieResponse.json();
+  
+  return nowPlayingMovieData.results;
+}
```

Comment:
API 호출 함수들의 중복되는 fetch 호출을 fetchMovies 함수로 추출하여 중복 코드를 제거하면 좋을 것 같습니다.

```suggestion
async function fetchMovies(url: string) {
  const response = await fetch(url, { cache: 'no-store' });

  if (!response.ok) {
    throw new Error(``);
  }

  const data = await response.json();
  return data.results;
}


export const getNowPlayingMovie = () => 
  fetchMovies(`https://api.themoviedb.org/3/movie/now_playing?api_key=${API_KEY}`);

```